### ISSUE TITLE
Copy package's latest version directly from the versions tab listing

### ISSUE THREAD

**Issue by Musaddiq625** [dart-lang/pub-dev#7736]
First of all thank you again for 'Copy Icon on listing' #7695 

I got a same feeling for this old versions tab too 😅
Because at this scenario, we only need the version

It would be very convenient for the developers to:

Search for a package
Go to detail page
Go to versions tab
Go to specific version's detail page ❌
Copy its latest version
Search for another package's old versions
Repeat steps 1 to 6

Currently, we need to go into the detail page of specific version from the versions tab listing just to copy the version number 🙏🏻

![image](https://github.com/dart-lang/pub-dev/assets/37911054/41d2b86b-70cc-4973-be0b-4b78178215b8)

**Comment by Musaddiq625** [comment]
@sigurdm

**Comment by sigurdm** [comment]
What should the copied text be?

"flutter_animate: ^4.4.1" will still end up with v.4.5.0...

What is the main use case here?

**Comment by Musaddiq625** [comment]
yes it will be like "flutter_animate: ^4.4.1"or "flutter_animate: ^4.2.0"
the use case here is to avoid the user to go to the detail page, allowing the user to copy the older version directly
